### PR TITLE
feat: Add Compare The Triplet HackerRank Js Coding Challenge

### PR DESCRIPTION
**Pull request by moshoodabdullahi**
# This PR adds compare the triplet hackerRank js coding challenge and here is the link to the [Compare The Triplets](https://www.hackerrank.com/challenges/compare-the-triplets) challenge

### CODE REVIEW

**Comment by Yinkusadel** [comment]
you should add the link to the challenge on your PR description

**Review by Yinkusadel — COMMENTED** [review]
you should add the link to the challenge in your PR description

**Review by Yinkusadel — DISMISSED** [review]
nice

File: .eslintrc.json [hunk 1]
```diff
@@ -12,6 +12,8 @@
   "plugins": ["prettier"],
   "rules": {
     "no-console": "warn",
+    "no-plusplus": "warn",
+    "no-unused-vars": "warn",
```
Comment (teckcipher) [inline .eslintrc.json]: Please remove this

**Review by teckcipher — APPROVED** [review]

File: hackerRank/compare-the-triplets/compare-the-triplets.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+const compareTriplets = (a, b) => {
+  let aliceScore = 0;
+  let bobScore = 0;
+
+  for (let i = 0; i < 3; i++) {
```
Comment (teckcipher) [inline hackerRank/compare-the-triplets/compare-the-triplets.js]: Let's chnage the `i++` to `i += 1`

**Review by teckcipher — CHANGES_REQUESTED** [review]

File: .eslintrc.json (hunk 1 above)
Comment (moshoodabdullahi) [reply .eslintrc.json]: Done

File: hackerRank/compare-the-triplets/compare-the-triplets.js (hunk 2 above)
Comment (moshoodabdullahi) [reply hackerRank/compare-the-triplets/compare-the-triplets.js]: Done

**Review by Yinkusadel — APPROVED** [review]
Ok

**Review by teckcipher — APPROVED** [review]

**Pull request merged**